main.go

```diff
@@ -7,7 +7,8 @@ import (
 	"github.com/segmentio/terraform-docs/internal/pkg/doc"
 	"github.com/segmentio/terraform-docs/internal/pkg/print"
 	"github.com/segmentio/terraform-docs/internal/pkg/print/json"
-	"github.com/segmentio/terraform-docs/internal/pkg/print/markdown"
+	markdown_content "github.com/segmentio/terraform-docs/internal/pkg/print/markdown/content"
```

Comment:
btw, I'm going to rename and `git mv` the `markdown/conent` to `markdown/document` at the very last for the purpose for code review be easier.

Reply:
Agreed, please continue with the renaming.

Reply:
Renamed and moved to correct package name.